src/main/java/com/github/fishio/SinglePlayerPlayingField.java

```diff
@@ -29,14 +26,18 @@
 	 * Creates the playing field for a single player.
 	 * 
 	 * @param fps
-	 *            the (target) framerate.
+	 *      the (target) framerate.
 	 * @param canvas
-	 *            the canvas to use, can be <code>null</code> to create one.
+	 *      the canvas to use, can be <code>null</code> to create one.
 	 * @param scene
-	 *            the scene to use (for registering keylisteners)
+	 *      the scene to use (for registering keylisteners)
+	 * @param width
+	 * 		the width of the playing field
+	 * @param height
+	 * 		the height of the playing field
```

Comment:
Good work on improving the layout of this javadocs. However, I think you forgot the identation of width and height?

<documentation